nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -538,7 +538,8 @@ class SearchParamDefaults:
     extracted = ParamDefault(
         default=[],
         title="Extracted metadata",
-        description="Controls which parts of the extracted metadata are serialized on search results",
+        description="[Deprecated] Please use GET resource endpoint instead to get extracted metadata",
```

Comment:
I think you should use this field instead of `description`:
https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.FieldInfo.deprecation_message